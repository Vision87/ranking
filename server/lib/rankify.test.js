const { elo, calculate } = require('./rankify')

describe('Search: data reducers', () => {
  const winnerTeam = {
    defender: {
      points: 1200
    },
    striker: {
      points: 1200
    },
    score: 6
  }

  const looserTeam = {
    defender: {
      points: 1200
    },
    striker: {
      points: 1200
    },
    score: 0
  }

  it('Should give a lot of points', () => {
    const result = calculate({
      teamHome: winnerTeam,
      teamAway: looserTeam
    })
    expect(result).toMatchSnapshot()
  })
})
