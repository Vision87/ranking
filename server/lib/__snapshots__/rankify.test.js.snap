// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search: data reducers Should give a lot of points 1`] = `
Object {
  "awayDefense": 1176,
  "awayDefense2": 1183,
  "awayStriker": 1176,
  "awayStriker2": 1183,
  "difference": 24,
  "difference2": 17,
  "hasCappotto": true,
  "hasHomeWin": true,
  "homeDefense": 1224,
  "homeDefense2": 1217,
  "homeStriker": 1224,
  "homeStriker2": 1217,
}
`;
